packages/cli/tests/unit/encodedFile.spec.ts

```diff
@@ -55,7 +55,8 @@ describe('EncodedFile', () => {
   });
 
   it('writes a 7-bit ASCII file', () => {
-    verifyUpdate('ascii', '\n');
+    const newlineChar = process.platform === 'win32' ? '\r\n' : '\n';
+    verifyUpdate('ascii', newlineChar);
```

Comment:
I don't understand how it now passes on Windows. The test file uses `\n`, so the test should now fail, right?

BTW, perhaps it would be a good idea to move those ascii tests into the per-platform section above.

Reply:
I'm not sure, but I've noticed that all files have `\r\n` by default in Windows. Even when I'm pulling them from GitHub, where they were generated with `\n`.

When I move the test into the per-platform section, I get these errors:
![image](https://user-images.githubusercontent.com/45714532/195431018-a0b0b326-24c3-48b5-b0d7-2e8ed292e1cb.png)

Reply:
Yup, that's probably the problem of your git configuration (autocrlf).